Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node
+
+RUN apt-get update -qq
+RUN apt-get install -yqq libprotobuf-dev
+RUN npm install -g --silent bower
+
+RUN mkdir /code
+WORKDIR /code
+
+ADD . /code/
+
+RUN npm install --silent
+RUN bower install --allow-root --silent
+
+EXPOSE 8081
+EXPOSE 3000
```

Comment:
You can now run it using this (after rebuilding the image):
```bash
docker run -d -p 3000:3000 -p 8081:8081 -it pogo
```

It's still possible to specify the command when running it in case you want to run something else:
```bash
docker run -d -p 3000:3000 -p 8081:8081 -it pogo npm update
```